library/src/test/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionBindingTest.java

```diff
@@ -175,4 +227,58 @@ public int getContentItemsTotal() {
         public void onBindItemViewHolder(RecyclerView.ViewHolder holder, int position) {
         }
     }
+
+    /**
+     * Override the inflate method in order to avoid mocking the LayoutInflater.
+     */
+    private static class AdapterImpl extends SectionedRecyclerViewAdapter {
+        @Override
+        View inflate(int layoutResourceId, ViewGroup parent) {
+            return dummyViewWithTag(layoutResourceId);
+        }
+    }
+
+    /**
+     * Helper class that implements getTag/setTag outside of Android environment.
+     */
+    private static class DummyView extends View {
+        private Object tag;
+
+        public DummyView(Context context) {
+            super(context);
+        }
+
+        @Override
+        public Object getTag() {
+            return tag;
+        }
+
+        @Override
+        public void setTag(Object tag) {
+            this.tag = tag;
+        }
+    }
+
+    /**
+     * @param tag Any integer value.
+     * @return A View that will have its tag set to the given value, to recognize it later on.
+     */
+    private static View dummyViewWithTag(int tag) {
+        View view = new DummyView(null);
+        view.setTag(tag);
```

Comment:
Can this be simplified as:
```java
View view = mock(View.class);
when(view.getTag()).thenReturn(tag);
```
and DummyView be removed?

Reply:
Indeed 👍